src/eth1spec/spec.py

```diff
@@ -333,6 +348,143 @@ def signing_hash(tx: Transaction) -> Hash32:
     )
 
 
+def header_hash(header: Header) -> Hash32:
```

Comment:
Since we're going with active naming, we should probably make this something like `calculate_header_hash` or `computer_header_hash`.

Reply:
I like `compute_header_hash`

Reply:
The reason why I chose this name was because the surrounding code had this name: 
https://github.com/quilt/eth1.0-specs/blob/609fdb060c6ba8101997e4e69328c840c78a135c/src/eth1spec/spec.py#L308

Reply:
I wrote some conventions down once here: https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/naming-things.md#prefixes